src/firefly/java/edu/caltech/ipac/astro/net/Simbad4Client.java

```diff
@@ -42,7 +42,7 @@
  */
 public class Simbad4Client {
 
-    public static final String SIMBAD_URL = "http://cdsweb.u-strasbg.fr/cgi-bin/nph-sesame/-oF/S?%s";
+    public static final String SIMBAD_URL = "http://cdsweb.u-strasbg.fr/cgi-bin/nph-sesame/-o/S?%s";
```

Comment:
Can you add documentation about what -o means.

Reply:
@lznakano you can point to the Sesame API used here http://cdsweb.u-strasbg.fr/doc/sesame.htx
('-o' the default ascii result not even documented in their docs )